### ISSUE TITLE
Contact form fieldset missing a legend element

### ISSUE THREAD

**Issue by CrispinF** [i7MEDIA/mojoportal#106]
Arising from an accessibility test on a mojoPortal site:

> A fieldset legend presents a description of the form elements within a fieldset and is especially useful to screen reader users. A legend should be provided when a higher level description is necessary for groups of check boxes, radio buttons, or other form controls.

Alternatively, consider removing the fieldset?
